### PR TITLE
Ran clippy and rustfmt

### PR DESCRIPTION
**Pull request by VictorKoenders**

### CODE REVIEW

**Comment by VictorKoenders** [comment]
Closes #4 and #2

File: src/undump.rs [hunk 1]
```diff
@@ -134,18 +131,13 @@ impl LoadState {
...
             // optimized later in the future, as well as the SyxString type, to
             // include a hash field.
             size -= 1;
-            if size < limits::SYX_MAXSHORTLEN {
-                // short string
-                return self.load_range(size);
-            } else { // long string
-                return self.load_range(size);
-            }
+            self.load_range(size)
```
Comment (RyanSquared) [inline src/undump.rs]: there's a reason I left that block there

File: src/undump.rs [hunk 2]
```diff
@@ -154,17 +146,18 @@ impl LoadState {
...
-                SyxType::TBOOLEAN => SyxValue::Bool(self.load::<u8>()? == 1),
-                SyxType::TNUMFLT => SyxValue::Number(self.load::<SyxNumber>()?),
-                SyxType::TNUMINT => SyxValue::Integer(self.load::<SyxInteger>()?),
-                | SyxType::TSHRSTR
-                | SyxType::TLNGSTR => SyxValue::String(self.load_string()?),
-                x => {
-                    return Err(format!("bad value for constant: {:?}", x));
-                }
-            });
+            proto
```
Comment (RyanSquared) [inline src/undump.rs]: nitpick: no

File: src/undump.rs [hunk 3]
```diff
@@ -101,8 +99,7 @@ impl LoadState {
             }
         };
         Ok(ret)
-        */
-    }
+        */    }
```
Comment (RyanSquared) [inline src/undump.rs]: nitpick: hell no

**Pull request merged**